arch-package/PKGBUILD

```diff
@@ -0,0 +1,72 @@
+# This is an example PKGBUILD file. Use this as a start to creating your own,
+# and remove these comments. For more information, see 'man PKGBUILD'.
+# NOTE: Please fill out the license field for your package! If it is unknown,
+# then please put 'unknown'.
+
+# Maintainer: Erik Bjäreholt <erik.bjareholt@gmail.com>
+pkgname=activitywatch
+pkgver='0.6.0a12'
+pkgrel=1
+epoch=
+pkgdesc="Log what you do on your computer. Simple, extensible, no third parties."
+arch=('i686' 'x86_64')
+url="https://github.com/ActivityWatch/activitywatch"
+license=('MPL2')
+groups=()
+depends=('python>=3.5.1')
+makedepends=('python>=3.5.1' 'python-setuptools')
+checkdepends=()
+optdepends=()
```

Comment:
Boilerplate I still might need. The PKGBUILD isn't ready and this is my first time packaging for Arch.